scripts/setupvars/setupvars.bat

```diff
@@ -37,10 +37,11 @@ if not "%1"=="" (
 
 :: OpenCV
 if exist "%INTEL_OPENVINO_DIR%\opencv\setupvars.bat" (
+set "OPENVINO_LIB_PATHS=%INTEL_OPENVINO_DIR%\opencv\bin;%OPENVINO_LIB_PATHS%"
```

Comment:
I think this should be handled on OpenCV side.

Reply:
As we discussed offline, OpenCV will be fixed separately (51531)